src/python/WMCore/Services/DBS/DBSErrors.py

```diff
@@ -111,3 +127,20 @@ def getReason(self):
         if isinstance(self.data, dict):
             return self.data['error']['reason']
         return ""
+
+def DBSErrors(dbsUrl):
```

Comment:
I would suggest not to mix the REST API call with the class dealing with a response object from the DBS server.

Perhaps a better place for it would be either in DBS3Reader:
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/Services/DBS/DBS3Reader.py
or DBSUtils, or DBSConcurrency... things are pretty much scattered out.

Reply:
fine I can move it there. To me it is more natural to keep errors with definition to error.

Reply:
ok, I moved code to DBSUtils.